app/src/test/java/nerd/tuxmobil/fahrplan/congress/dataconverters/LectureExtensionsTest.kt

```diff
@@ -104,10 +105,39 @@ class LectureExtensionsTest {
         val lecture = Lecture("")
         lecture.date = "2015-08-13"
         lecture.day = 3
-        val dateInfo = DateInfo(3, "2015-08-13")
+		val dateInfo = DateInfo(3, SystemClock("2015-08-13"))
         assertThat(lecture.toDateInfo()).isEqualTo(dateInfo)
     }
 
+    @Test
+    fun toDayRanges() {
+        val lecture0 = Lecture("")
+        lecture0.date = "2019-08-02"
+        lecture0.day = 2
+
+        val lecture1 = Lecture("")
+        lecture1.date = "2019-08-01"
+        lecture1.day = 1
+
+        val lecture2 = Lecture("")
+        lecture2.date = "2019-08-01"
+        lecture2.day = 1
```

Comment:
# Naming
The test becomes easier to understand if you rename `lecture2` to `lecture1Copy` since the objects are equal.